tools/webpack/shared.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+const postcss = require( 'postcss' );
+const { join } = require( 'path' );
+
+const baseDir = join( __dirname, '../../' );
+
+const baseConfig = ( env ) => {
+	const mode = env.environment;
+
+	const config = {
+		target: 'browserslist',
+		mode,
+		optimization: {
+			moduleIds: mode === 'production' ? 'deterministic' : 'named',
```

Comment:
Can you explain why `moduleIds` is different in webpack 5? Did they change the name?

Reply:
Correct. I found an answer: https://webpack.js.org/configuration/optimization/#optimizationmoduleids.

Reply:
Yes, they have deprecated `hashed` in favor of `deterministic`. The the migration guide they mention that you should consider to remove `moduleIds` if you use that (https://webpack.js.org/migrate/5#clean-up-configuration). Need to read trough https://core.trac.wordpress.org/ticket/53192 and understand why it was added in the first place.